db scripts/add_static_fields_to_investigation.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE public.investigation
+    ADD COLUMN is_return_sick boolean NOT NULL DEFAULT false,
+    ADD COLUMN previous_disease_start_date timestamp with time zone,
+    ADD COLUMN is_vaccinated boolean NOT NULL DEFAULT false,
+    ADD COLUMN vaccination_effective_from timestamp with time zone,
+    ADD COLUMN is_suspicion_of_mutation boolean NOT NULL DEFAULT false,
```

Comment:
If the default option is false how would those fields ever be null? (Defined in client side)